packages/ember-headless-form/src/-private/components/control/select.gts

```diff
@@ -43,17 +45,38 @@ export interface HeadlessFormControlSelectComponentSignature {
   Blocks: {
     default: [
       {
-        Option: WithBoundArgs<typeof OptionComponent, 'selected'>;
+        Option: WithBoundArgs<
+          typeof HeadlessFormControlSelectOptionComponent,
+          'selected'
+        >;
       }
     ];
   };
 }
 
 export default class HeadlessFormControlSelectComponent extends Component<HeadlessFormControlSelectComponentSignature> {
-  OptionComponent = OptionComponent;
-
   @action
   handleInput(e: Event | InputEvent): void {
     this.args.setValue((e.target as HTMLSelectElement).value);
   }
+
+  <template>
+    <select
+      name={{@name}}
+      value={{@value}}
+      id={{@fieldId}}
+      aria-invalid={{if @invalid "true"}}
+      aria-describedby={{if @invalid @errorId}}
+      ...attributes
+      {{on "input" this.handleInput}}
+    >
+      {{yield
+        (hash
+          Option=(component
+            HeadlessFormControlSelectOptionComponent selected=@value
```

Comment:
don't need `ensure-safe-component` anymore?

Reply:
Nope! :tada:

With strict mode templates, the `{{component}}` helper is also more strict, not allowing a string based lookup anymore. This makes sense as you would have explicit imports anyway. And by that, Embroider can safely assume that whatever is passed to `{{component  myComponent}}` as the first argument is already something that has been imported _somewhere_ (and as such it is already "seen" by Embroider/webpack and added to the bundle), so it does not need to care about this being potentially unsafe anymore. I also learned about this more recently...